tools/logger/convert.c

```diff
@@ -327,6 +327,10 @@ int convert(struct convert_config *config) {
 			fprintf(stderr, "Error: abi version in %s file "
 				"does not coincide with abi version used "
 				"by logger.\n", config->version_file);
+			fprintf(stderr, "logger ABI Version is %d\n",
+				SOF_ABI_VERSION);
```

Comment:
@ranj063 
why not use SOF_ABI_VERSION_MAJOR(src/include/uapi/abi.h) and other macro to parse a more readable output?

Reply:
Best to also include MINOR and PATCH versions and ABI version of ldc file too. That way users can tell if ldc or logger is too old.